docs/namespaces.md

```diff
@@ -74,4 +74,8 @@ will all use the default namespace, which is simply named "`default`".
 ```
 
 You can also use the `CONTAINERD_NAMESPACE` environment variable to specify the default namespace to use for
-any of the `ctr` client commands.
+any of the `ctr` client commands.  
+>To know more about different Namespaces packages click below links :  
+>[`containerd Namespaces`](https://pkg.go.dev/github.com/containerd/containerd/v2/pkg/namespaces).  
+>[`Kubernetes Namespaces`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).  
```

Comment:
Kubernetes namespaces are independent of containerd namespaces.  Including it without context seems confusing.

I think we should remove this link.

Reply:
I have removed all the links please review it & approve it,

Thank you